tests/copy-paste.js

```diff
@@ -457,7 +457,7 @@ define([
                 ["id", "type", "calculateAttr"],
                 ["/radius", "DataBindOnly", "42"],
                 ["/pi", "DataBindOnly", "3.1415"],
-                ["/circumference", "DataBindOnly", "2 * #form/pi * #form/radius"],
+                ["/circumference", "DataBindOnly", "2 * `#form/pi` * `#form/radius`"],
```

Comment:
How hard would it be to use the same convention here as in XML?
- prefix invalid xpath expressions with `#invalid/xpath ` and escape hashtags
- do not escape hashtags in valid xpath expressions